app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -273,11 +273,8 @@ class HomeScreenRobot {
 
         fun togglePrivateBrowsingMode() {
             mDevice.findObject(UiSelector().resourceId("$packageName:id/privateBrowsingButton"))
-                .waitForExists(
-                    waitingTime
-                )
-            privateBrowsingButton()
-                .perform(click())
+                .waitForExists(waitingTime)
```

Comment:
you can replace here with privateBrowsingButton.waitForExists(waitingTime)

Reply:
Left it as it was, only made a minor refactoring.